baltic-honeybadger/2018/bitcoin-custody.es.md

```diff
@@ -6,8 +6,6 @@ categories: ['conference']
 tags: ['custodia', 'reglamento']
 ---
 
-Bryan Bishop
```

Comment:
Bryan Bishop is also the speaker for this.

Reply:
Alright, noted. I will fix